app/models/user.rb

```diff
@@ -34,7 +36,24 @@ class User < ActiveRecord::Base
   has_many :status
 
   accepts_nested_attributes_for :status
-  scope     :mail_receiver, -> { where(receive_mailings: true) }
+  scope :mail_receiver, -> { where(receive_mailings: true) }
+  scope :project_filter, -> (project_id) {
+    joins(:follows)
+    .where(
+      follows: {
+        blocked: false,
+        followable_id: project_id,
+        followable_type: 'Project',
+        follower_type: 'User'
+      }
+    )
+  }
+  scope :timezone_filter, -> (degrees) {
+    where("users.longitude BETWEEN ? AND ?", degrees[0], degrees[1])
+  }
+  scope :allow_to_display, -> { where(display_profile: true) }
+  scope :by_create, -> { order(:created_at) }
```

Comment:
`by_create` doesn’t really say much about what it is doing, a better name would be `order_by_created_at`, but at this point, it might just be better to use `order(:created_at)` rather than a scope!